examples/validate_relative_binding.py

```diff
@@ -8,43 +8,40 @@
 # 4) Release the restraints attached to the blocker
 # Note that 2) and 4) are done directly via an endpoint correction.
 
+import argparse
```

Comment:
What's the reason for moving these 3 above `os` and `pickle`?

Reply:
Furthermore, this seems to break the comment into two sections as opposed to keeping them together

Reply:
Good catch, I was also puzzled by this one. I think the comment is considered to be attached to the following import line (in this case `import os`), and since `ensure_newline_before_comments` is enabled (by `profile = "black"`), isort inserts an extra newline before the comment.